src/main/java/com/icthh/xm/ms/entity/repository/backend/StorageRepository.java

```diff
@@ -45,7 +44,13 @@ private String store(InputStream stream, Integer size, String contentType, Strin
             }
             String filename = UUID.randomUUID().toString() + "." + FilenameUtils.getExtension(name);
             amazonS3Template.save(filename, stream);
-            IOUtils.closeQuietly(stream);
+
+            try {
+                if (stream != null) {
+                    stream.close();
+                }
+            } catch (IOException ignore) {
```

Comment:
maybe better add debug or trace log, or something like this.

Reply:
done